build.fsx

```diff
@@ -125,12 +127,18 @@ Target "RestoreDotNetCore" (fun _ ->
 
 Target "BuildDotNetCore" (fun _ ->
     !! "./**/project.json"
+    -- "./build/**"
     |> Fake.DotNetCli.Build id
 )
 
 Target "UnitTestsDotNetCore" (fun _ ->
-    !! "./Octokit.Next.Tests"
-    |> Fake.DotNetCli.Test id
+    [ "./Octokit.Tests"
+      "./Octokit.Tests.Conventions" ]
+    |> Seq.iter (fun d ->
+        Fake.DotNetCli.Test (fun p ->
```

Comment:
Fair point - I just pulled it down to do a `.\build` but I guess that's now handled by something else...